packages/esm-patient-chart-app/src/visit-header/visit-header.component.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import capitalize from 'lodash-es/capitalize';
+import {
+  Button,
+  DefinitionTooltip,
+  Header,
+  HeaderContainer,
+  HeaderGlobalAction,
+  HeaderGlobalBar,
+  HeaderMenuButton,
+} from '@carbon/react';
+import { CloseFilled } from '@carbon/react/icons';
+import {
+  age,
+  ConfigurableLink,
+  useAssignedExtensions,
+  useLayoutType,
+  usePatient,
+  useVisit,
+  navigate,
+} from '@openmrs/esm-framework';
+import { launchPatientWorkspace } from './workspaces';
+import VisitHeaderSideMenu from './visit-header-side-menu.component';
+import styles from './visit-header.scss';
+
+const PatientInfo = ({ patient, isTabletView }) => {
+  const name = `${patient?.name?.[0].given?.join(' ')} ${patient?.name?.[0].family}`;
```

Comment:
I think this can be changed to `patient.displayName` for better localization

Reply:
Hmm you're right. It looks like this isn't available anymore with fhir api? I can see it through REST endpoint for say visit - but even then its not always available
![Screen Shot 2022-09-07 at 9 51 12 AM](https://user-images.githubusercontent.com/5445264/188937056-678857ac-0173-46a9-831c-6adc6a5c6972.png)

Reply:
Display is not available with the FHIR, its a property of the REST